metabase/metabase
# Align combo chart ticks, axes names

### Description

Aligns axis ticks and labels of the combo chart.

Closes https://github.com/metabase/metabase/issues/35776

### How to verify

1. Create combo charts with different combinations of axes to verify long ticks, short ticks, rotated X-axis ticks, disabled axes, disabled labels
2. Add them on a dashboard
3. Sent a test subscription to verify they look as expected and layouts are correct

### Demo

#### Show axes, show axes names
Dashcard:
<img width="552" alt="Screenshot 2023-11-14 at 9 33 00 PM" src="https://github.com/metabase/metabase/assets/14301985/e4031525-acdc-4a67-b752-5ecf5bc9b082">
Static viz:
<img width="590" alt="Screenshot 2023-11-14 at 9 33 43 PM" src="https://github.com/metabase/metabase/assets/14301985/1debcadb-2705-45de-9544-44421d0af8b1">

#### Hide axes, show axes names
Dashcard:
<img width="553" alt="Screenshot 2023-11-14 at 9 33 06 PM" src="https://github.com/metabase/metabase/assets/14301985/14e879c7-7c7a-4a5b-8ac8-52e79c228b81">

Static viz:
<img width="605" alt="Screenshot 2023-11-14 at 9 33 49 PM" src="https://github.com/metabase/metabase/assets/14301985/78b13c35-637b-4ddc-b747-9b4754e7989f">

#### Show axes, hide axes names
Dashcard:
<img width="555" alt="Screenshot 2023-11-14 at 9 33 13 PM" src="https://github.com/metabase/metabase/assets/14301985/8bc319b0-c2ef-41c6-9b9c-c884e86a5db2">

Static viz:
<img width="592" alt="Screenshot 2023-11-14 at 9 33 57 PM" src="https://github.com/metabase/metabase/assets/14301985/4d86ecb8-06d0-4352-affb-1de3a452a006">

#### Hide axes, hide axes names
Dashcard:
<img width="551" alt="Screenshot 2023-11-14 at 9 33 17 PM" src="https://github.com/metabase/metabase/assets/14301985/16c913f4-8b25-4c6b-9b97-22322a4fbf37">

Static viz:
<img width="602" alt="Screenshot 2023-11-14 at 9 34 04 PM" src="https://github.com/metabase/metabase/assets/14301985/84f589a0-322f-4f21-a395-6e8a010981f1">

### Checklist
